### ISSUE TITLE
Milvus server can not be connected after milvus pod restart

### ISSUE THREAD

**Issue by ThreadDao** [milvus-io/milvus#5696]
**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**
1. the milvus server crashed
2. the k8s restart milvus pod automatically
```
NAME                                                  READY   STATUS    RESTARTS   AGE
milvus-testing-etcd-0                                 1/1     Running   0          27h
milvus-testing-milvus-ha-standalone-c676857c5-7pqm9   1/1     Running   1          27h
milvus-testing-minio-7cfb4db88-4td9r                  1/1     Running   0          27h
```
3. Milvus server can not be connected
```
server is not healthy, please try again later 
```
The server log:
```
8112 2021-06-09T09:37:27.086695533Z stderr F 2021-06-09 09:37:27.086521 I | retry func error: WaitForComponentStates, not meet, MasterService current state:0. attempts #998854 after 1s.
8113 2021-06-09T09:37:27.343055092Z stdout F [2021/06/09 09:37:27.342 +00:00] [DEBUG] [impl.go:1495] [RegisterLink] [role=] ["state code of proxynode"=Abnormal]
8114 2021-06-09T09:37:27.605253304Z stdout F [2021/06/09 09:37:27.604 +00:00] [DEBUG] [master_service.go:1196] [GetComponentStates] ["State Code"=Initializing]
8115 2021-06-09T09:37:27.605559671Z stderr F 2021-06-09 09:37:27.605385 I | retry func error: WaitForComponentStates, not meet, MasterService current state:0. attempts #998853 after 1s.
8116 2021-06-09T09:37:27.82787226Z stdout F [2021/06/09 09:37:27.827 +00:00] [DEBUG] [master_service.go:1196] [GetComponentStates] ["State Code"=Initializing]
8117 2021-06-09T09:37:27.828999521Z stderr F 2021-06-09 09:37:27.828136 I | retry func error: WaitForComponentStates, not meet, MasterService current state:0. attempts #998855 after 1s.
8118 2021-06-09T09:37:28.087768219Z stdout F [2021/06/09 09:37:28.087 +00:00] [DEBUG] [master_service.go:1196] [GetComponentStates] ["State Code"=Initializing]
8119 2021-06-09T09:37:28.088026523Z stderr F 2021-06-09 09:37:28.087831 I | retry func error: WaitForComponentStates, not meet, MasterService current state:0. attempts #998853 after 1s.
8120 2021-06-09T09:37:28.347040851Z stdout F [2021/06/09 09:37:28.346 +00:00] [DEBUG] [impl.go:1495] [RegisterLink] [role=] ["state code of proxynode"=Abnormal]
8121 2021-06-09T09:37:28.606210074Z stdout F [2021/06/09 09:37:28.605 +00:00] [DEBUG] [master_service.go:1196] [GetComponentStates] ["State Code"=Initializing]
8122 2021-06-09T09:37:28.606464865Z stderr F 2021-06-09 09:37:28.606297 I | retry func error: WaitForComponentStates, not meet, MasterService current state:0. attempts #998852 after 1s.
8123 2021-06-09T09:37:28.82922984Z stdout F [2021/06/09 09:37:28.828 +00:00] [DEBUG] [master_service.go:1196] [GetComponentStates] ["State Code"=Initializing]
8124 2021-06-09T09:37:28.829672849Z stderr F 2021-06-09 09:37:28.829451 I | retry func error: WaitForComponentStates, not meet, MasterService current state:0. attempts #998854 after 1s.
8125 2021-06-09T09:37:29.089007542Z stdout F [2021/06/09 09:37:29.088 +00:00] [DEBUG] [master_service.go:1196] [GetComponentStates] ["State Code"=Initializing]
8126 2021-06-09T09:37:29.089339487Z stderr F 2021-06-09 09:37:29.089114 I | retry func error: WaitForComponentStates, not meet, MasterService current state:0. attempts #998852 after 1s.
8127 2021-06-09T09:37:29.350819079Z stdout F [2021/06/09 09:37:29.350 +00:00] [DEBUG] [impl.go:1495] [RegisterLink] [role=] ["state code of proxynode"=Abnormal]
8128 2021-06-09T09:37:29.607508047Z stdout F [2021/06/09 09:37:29.607 +00:00] [DEBUG] [master_service.go:1196] [GetComponentStates] ["State Code"=Initializing]
8129 2021-06-09T09:37:29.607880997Z stderr F 2021-06-09 09:37:29.607689 I | retry func error: WaitForComponentStates, not meet, MasterService current state:0. attempts #998851 after 1s.
8130 2021-06-09T09:37:29.829858297Z stdout F [2021/06/09 09:37:29.829 +00:00] [DEBUG] [master_service.go:1196] [GetComponentStates] ["State Code"=Initializing]
8131 2021-06-09T09:37:29.829946952Z stderr F 2021-06-09 09:37:29.829896 I | retry func error: WaitForComponentStates, not meet, MasterService current state:0. attempts #998853 after 1s.
8132 2021-06-09T09:37:30.089643382Z stdout F [2021/06/09 09:37:30.089 +00:00] [DEBUG] [master_service.go:1196] [GetComponentStates] ["State Code"=Initializing]
8133 2021-06-09T09:37:30.089669568Z stderr F 2021-06-09 09:37:30.089630 I | retry func error: WaitForComponentStates, not meet, MasterService current state:0. attempts #998851 after 1s.                                                                                                                                                                               
```

**Steps/Code to reproduce behavior**
Follow this [guide](http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) to craft a minimal bug report. This helps us reproduce the issue you're having and resolve the issue more quickly.

**Expected behavior**

**Method of installation**
- [ ] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Software conditions (OS, CPU, GPU, Memory)

- Milvus version (master or released version)
 pymilvus              2.0a1.dev34
 standalone

**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
paste-file-content-here
```

**Screenshots**

**Additional context**

**Comment by DragonDriver** [comment]
Same as: #5901

**Comment by DragonDriver** [comment]
Already fixed by #6073 , please retest this.

/unassign @DragonDriver 
/assign @ThreadDao

**Comment by ThreadDao** [comment]
After chaos testing, this issue seems to have been fixed.
